src/main/resources/Documentation/about.md

```diff
@@ -1,41 +1,55 @@
-TODO: Add Plugin introduction
+Eiffel Gerrit plugin is may be used to send Eiffel events when changes occurs in in a project in Gerrit.
+This plugin simplifies communication in continuous integration and delivery pipelines and helps keeping
+source code management traceability.
 
 For more Eiffel information follow this link:
 <https://github.com/eiffel-community/eiffel-gerrit-plugin>
 
+#####Eiffel Events:
+
+Version 0.0.1
+:   SourceChangeCreatedEvent
+    SourceChangeSubmittedEvent
+
 Parameters
 ----------
 
 Project Configuration:
 
 eiffel-integration.enabled
-:   Plugin state. if set to `true`, send message Eiffel message for every commit submitted for this repository.
-    Plugin considered disabled if this option is not defined or set to `false`.
+:   If set to `true` the plugin will send eiffel events.
+    Is set to `false` or is not defined no eiffel events will be sent.
```

Comment:
'If set to `false` or if not defined' and also 'Eiffel'

Reply:
Done in next commit.

Reply:
Done.











....